### DISCUSSION TITLE
React to sounds, like ai model for them or similar?

### DISCUSSION THREAD

**Discussion by gurkburk76** [blakeblackshear/frigate#7095] [Ask A Question]
So i just got a reolink poe doorbell that's great, it interacts with home assistant just the way i like it to adn all that jazz.
But, for some reason the postmen/women are either dumb or just refuse to use it and instead persist on *knocking* on the door, yeah much good that does me when i'm in the garage (i'll set up a doorbell there soon enough, not that it will help if they insist on knocking though...) or or the second floor/basement.
So, my tldr question is: is there any way i can go about things to reqognize a knocking sounds and have it play a custom reply?

**Comment by NickM-27** [comment]
Audio detection will be supported in 0.13

**Comment by gurkburk76** [comment]
That's excelent.. now i just have to figure out if it's possible to make my doorbell "speak", using ha so i guess that would be involved somehow.
It seems like it's got 2-way audio recently so i guess it should be doable, has anyone else tried this ? :)

**Reply by NickM-27** [reply]
Yes, that is supported

**Comment by gurkburk76** [comment]
Do,,you have an example on how this could look like in yaml for ha possibly? :)
Would be easy if the doorbell presented itself as a media_player.* but it dosen't.
